packages/api/src/main/enforce-env-vars/index.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright IBM Corp. 2022, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { getRunMode, PRODUCTION } from '../run-mode'
```

Comment:
When @jdharvey-ibm's PR gets merged, this will need updated to `PROD`. Just an FYI.